packages/app/src/cli/services/dev/update-extension.ts

```diff
@@ -55,7 +57,7 @@ export async function updateExtensionDraft({
   if (mutationResult.extensionUpdateDraft?.userErrors?.length > 0) {
     const errors = mutationResult.extensionUpdateDraft.userErrors.map((error) => error.message).join(', ')
     stderr.write(`Error while updating drafts: ${errors}`)
-  } else {
+  } else if (extension.displayDraftUpdateSuccesMessage()) {
     outputInfo(`Draft updated successfully for extension: ${extension.localIdentifier}`, stdout)
```

Comment:
I'm wondering if saying `App config updated successfully` is a bit of a lie at this point. Yes we're updating the drafts, but for most of those configuration modules, they're not testable via `dev`, only some are.

Either we 
- Hide that we're updating the drafts for these things so that people don't get the incorrect impression that we updated their production state app configuration, or so that people don't think they can see those drafts like other extensions. 
- Don't mention we're updating configuration modules at all 
- Only mention the ones where they can be tested via `dev` 
  - I don't love this option since the modules that can be tested can't be tested in their entirety 
  - If we show things on a per field basis that might get large and complicated for not a lot of benefit

Reply:
+1 to @pt2pham - I'd say let's **not** tell the user that config modules have had their drafts updated. For now. Once all config is testable, we can say something like "Draft updated successfully for app configuration"

Reply:
ok. So I should remove any reference to modification in the toml right?
- Success message when the draft content is updated
- Error message if the user introduces an error in the toml
Right?

Reply:
Yes makes sense